src/fides/api/service/connectors/datahub_connector.py

```diff
@@ -0,0 +1,35 @@
+from typing import Generic, Optional
+
+from datahub.ingestion.graph.client import DataHubGraph, DataHubGraphConfig
+from loguru import logger
+
+from fides.api.models.connectionconfig import ConnectionConfig, ConnectionTestStatus
+from fides.api.schemas.connection_configuration.connection_secrets_datahub import (
+    DatahubSchema,
+)
+from fides.api.service.connectors.base_connector import DB_CONNECTOR_TYPE
+
+
+class DatahubConnector(Generic[DB_CONNECTOR_TYPE]):
+
+    def __init__(self, configuration: ConnectionConfig):
+        self.configuration = configuration
+        self.config = DatahubSchema(**configuration.secrets or {})
+        # TODO: use token for authentication
+        self.datahub_client = DataHubGraph(
+            DataHubGraphConfig(server=str(self.config.datahub_server_url))
+        )
```

Comment:
right now the datahub we're running doesn't require any token, I'll address this in a separate ticket